python/tests/conftest.py

```diff
@@ -53,6 +58,15 @@ def remove_test_dashboards(sdk: mtds.LookerSDK, dashboards):
                 sdk.delete_dashboard(cast(str, dashboard.id))
 
 
+@pytest.fixture(scope="function")
+def remove_test_looks(sdk: mtds.LookerSDK, looks):
+    for l in looks:
+        search_results = sdk.search_looks(title=l["title"])
+        if len(search_results) > 0:
+            for look in search_results:
+                sdk.delete_look(cast(int, look.id))
```

Comment:
If we add an additional look to data.yml, will this loop actually work to match all looks and delete them?

Reply:
Yes. `looks` on line 62 is a list containing all looks defined in `data.yml`. As long as each look has a title, we'll be able to find it and delete it.

Reply:
sorry, I missed the iterator on line 63!